adapters/cisco_ftd_rest/conf/device.properties

```diff
@@ -0,0 +1,7 @@
+model.id = 2070001
+model.name = FTD
+manufacturer.id = 1
+manufacturer.name = Cisco
+manufacturer.category = Security
```

Comment:
all the other Cisco are in category Network

Reply:
Oh I saw that and I fixed on my MSA but forgot in source !

Reply:
but we can put FTD in Security, I think @ubihgu may have an opinion on this